server.go

```diff
@@ -370,11 +374,20 @@ func main() {
 	if err != nil {
 		log.Fatalf("Failed to listen: %v", err)
 	}
+	liquidEsploraBaseURL, err := url.Parse(os.Getenv("LIQUID_ESPLORA_API_BASE_URL"))
+	if err != nil {
+		log.Fatalf("Failed to parse %v: %v", os.Getenv("LIQUID_ESPLORA_API_BASE_URL"), err)
+	}
+
+	proxy := httputil.NewSingleHostReverseProxy(liquidEsploraBaseURL)
 	mux := http.NewServeMux()
 	mux.HandleFunc("/fees/v1/btc-fee-estimates.json", func(w http.ResponseWriter, req *http.Request) {
 		w.Header().Set("Content-Type", "application/json")
 		w.Write([]byte(feeEstimates))
 	})
+	mux.HandleFunc(fmt.Sprint("POST ", liquidAPIPrefix, "/tx"), liquid.BroadcastHandler(liquidAPIPrefix, proxy, liquidEsploraBaseURL))
+	mux.HandleFunc(fmt.Sprint("GET ", liquidAPIPrefix, "/mempool"), liquid.MempoolHandler(liquidAPIPrefix, proxy, liquidEsploraBaseURL))
+	mux.HandleFunc(fmt.Sprint("GET ", liquidAPIPrefix, "/mempool/"), liquid.MempoolHandler(liquidAPIPrefix, proxy, liquidEsploraBaseURL))
```

Comment:
Here we need the transaction of a specific script hash: https://github.com/blockstream/esplora/blob/master/API.md#get-scripthashhashtxs
e.g:
GET /scripthash/:hash/txs